pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -101,6 +105,11 @@ func (c TargetConfigController) sync(ctx context.Context, syncCtx factory.SyncCo
 		return fmt.Errorf("TargetConfigController missing env var values")
 	}
 
+	backupVars := c.backupVarGetter.GetBackupVars()
+	if len(backupVars) == 0 {
+		return fmt.Errorf("TargetConfigController missing backup var values")
+	}
```

Comment:
it's fine if they're missing, no need to error the controller here

Reply:
done 👍🏽